src/devices/acurite.c

```diff
@@ -408,13 +412,77 @@ similar RF encoding and data format:
 - 592TXR temperature and humidity sensor
 - 5-n-1 weather station
 - 6045M Lightning Detector with Temperature and Humidity
+- Atlas
 
     CC RR IIII | IIII IIII | pBMMMMMM | pxxWWWWW | pWWWTTTT | pTTTTTTT | pSSSSSSS
     C:2d R:2d ID:12d 1x BATT:1b TYPE:6h 1x ?2b W:5b 1x 3b T:4b 1x 7b S: 1x 7d
 
 @todo - refactor, move 5n1 and txr decoding into separate functions.
 @todo - TBD Are parity and checksum the same across these devices?
         (opportunity to DRY-up and simplify?)
+
+Acurite Atlas:
+
+| Reading           | Operating Range               | Reading Frequency | Accuracy |
+| ---               | ---                           | ---        | ---             |
+| Temperature Range | -40 to 158°F (-40 to 70°C)    | 30 seconds | ± 1°F |
+| Humidity Range    | 1-100% RH                     | 30 seconds | ± 2% RH |
+| Wind Speed        | 0-160 mph (0-257 km/h)        | 10 seconds | ± 1 mph ≤ 10 mph, ± 10% > 10 mph |
+| Wind Direction    | 360°                          | 30 seconds | ± 3° |
+| Rain              | .01 inch intervals (0.254 mm) | 30 seconds | ± 5% |
+| UV Index          | 0 to 15 index                 | 30 seconds | ± 1 |
+| Light Intensity   | to 120,000 Lumens             | 30 seconds | n/a |
+| Lightning         | Up to 25 miles away (40 km)   | 10 seconds | n/a |
+
+The Atlas reports direction with an AS5600 hall effect sensor, it has 12-bit resolution according to the spec sheet. https://ams.com/as5600
+
+Acurite Atlas Message Type Format:
+
+Message Type 0x25 (Wind Speed, Temperature, Relative Humidity, ???)
+
+    Byte 1   Byte 2   Byte 3   Byte 4   Byte 5   Byte 6   Byte 7   Byte 8   Byte 9   Byte 10
+    cc??ssdd dddddddd pb011011 pWWWWWWW pWTTTTTT pTTTTTTT pHHHHHHH pXXXXXXX pXXXXXXX kkkkkkkkk
+
+Note: 13 bits for Temp is too much, should only be 11 bits.
+
+Message Type 0x26 (Wind Speed, Wind Vector, Rain Counter, ???)
+
+    Byte 1   Byte 2   Byte 3   Byte 4   Byte 5   Byte 6   Byte 7   Byte 8   Byte 9   Byte 10
+    cc??ssdd dddddddd pb011100 pWWWWWWW pW?VVVVV pVVVVV?? pRRRRRRR pXXXXXXX pXXXXXXX kkkkkkkkk
+
+    CHANNEL:2b xx ~SEQ:2d ~DEVICE:10d xx ~TYPE:6h SPEED:x~7bx~1b DIR:x~5bx~5bxx x~7b x~7b x~7b CHK:8h
+
+Note: 10 bits for Vector is too much, should only be 9 bits.
```

Comment:
Minor nit - if both the code and comment bit fields have been updated this should probably be removed.